textile/fabric_pretreatment/doctype/pretreatment_order/pretreatment_order.py

```diff
@@ -248,10 +248,10 @@ def start_pretreatment_order(self):
 
 		# Sales Order
 		if flt(self.per_ordered) < 100 and not self.is_internal_customer:
-			sales_order = _make_sales_order(self.name, ignore_permissions=True)
+			sales_order = _make_sales_order(self.name, ignore_permissions=ignore_permissions)
```

Comment:
This should always be True. Check permissions separately before creating document